# Includes penguin label name
string label
# Engaging, scanning, disengaging, or complete
string status

